web/src/components/organisms/Asset/Asset/hooks.ts

```diff
@@ -69,11 +70,20 @@ export default (assetId?: string) => {
     setIsModalVisible(false);
   }, []);
 
+  const handleCollapsed = useCallback(
+    (value: boolean) => {
+      setCollapsed(value);
+    },
+    [setCollapsed],
+  );
```

Comment:
Collapsing is just in one direction. Perhaps `handleToggleMenu`, `handleToggleCommentMenu`, etc is better. Not sure which menu, sidebar, etc this is collapsing/opening

Reply:
You are right, changed to `toggle` in the 2 places (`Asset/hooks.ts` and `AssetList/hooks.ts`) where the same logic exists.